File: Move Images to RAW.m
(*Get the working directory via input
Expect the current directory to have a directory named Images, which contaiens a folder for each image day(which should be inputed), containing a bunch of folders*)
MainDirec = SetDirectory[Directory[] <> "\\Images\\" <> ToString[Input["Input the directory path to folder of images to anylyze from the Images folder: "]]];

Print["Loading Files..."]
(*find folders in the current directory*)
Folders = Flatten[FileNames[All, MainDirec]]
NumFolders = Length[Folders]

(*get all image files*)
Files = ParallelTable[Flatten[Select[FileNames[All, Folders[[i]]], StringMatchQ[#, __ ~~ "tif"] &]], {i, NumFolders}]
NumImages = ParallelTable[Length[Files[[i]]], {i, NumFolders}]

(*Get all image file names*)
ImageFileNames = ParallelTable[Flatten[FileNames[All, Folders[[i]]]], {i, NumFolders}]
ImageBaseFileNames =  ParallelTable[FileBaseName[ImageFileNames[[i, j]]], {i, NumFolders}, {j, NumImages[[i]]}]

Print["Creating RAW folder and moving images..."]
ParallelTable[Quiet[CreateDirectory[Folders[[i]] <> "\\RAW"]], {i, NumFolders}] 
ParallelTable[CopyFile[Files[[i, j]], Folders[[i]] <> "\\RAW\\" <> ImageBaseFileNames[[i, j]] <> ".tif"], {i, NumFolders}, {j, NumImages[[i]]}]
ParallelTable[DeleteFile[Folders[[i]] <> "\\" <> ImageBaseFileNames[[i, j]] <> ".tif"], {i, NumFolders}, {j, NumImages[[i]]}]

File: move_images_to_raw.py
import os

#assumes the following path format
#\Nanotube Image Analysis
#   this file.py
#   \Images
#       \Date1
#       \Date2
#           \images folder
#           \another images folder
#               \RGB Stacks jpg
#                   image 1
#                   image 2

#change and get current working directory (cwd)
os.chdir('{}\Images\{}'.format(os.getcwd(), input('Input the name of the image day folder: ')))
cwd = os.getcwd()

base_folders = os.listdir(cwd)

for folder in base_folders:
    #get list of image file names
    image_files = (os.listdir('{}\{}'.format(cwd, folder)))
    #make a new directory
    os.mkdir('{}\{}\RAW'.format(cwd, folder))
    for image_file_name in image_files:
        os.rename('{}\{}\{}'.format(cwd, folder, image_file_name), '{}\{}\RAW\{}'.format(cwd, folder, image_file_name))